master/internal/api_trials.go

```diff
@@ -77,3 +79,22 @@ func (a *apiServer) TrialLogs(
 		time.Sleep(batchWaitTime)
 	}
 }
+
+func (a *apiServer) KillTrial(ctx context.Context, req *apiv1.KillTrialRequest,
```

Comment:
non-blocking: the formatting here

Reply:
I feel like the go formatter is weird. if I don't want to do this I'll have to leave the return type definition parenthesis open on line 83.

Reply:
what style do you suggest

Reply:
yep.